Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/zip.test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+./zip.sh
+unzip ./Package/IncidentProducer.zip -d IncidentProducer.output1
+git checkout ./Package/IncidentProducer.zip
+unzip ./Package/IncidentProducer.zip -d IncidentProducer.output2
+kdiff3 IncidentProducer.output1 IncidentProducer.output2
```

Comment:
What's the expectation of this file having kdiff3? To see what are the differences between the current and the previous versions? Also, is kdiff3 expected to be in all the systems?

Reply:
yes, I want to compare, the zip file created by my shell and the zip created by your IDE...

Reply:
yes, to compare the unzip folders.
kdiff3 is easy to install in both macos, Linux.